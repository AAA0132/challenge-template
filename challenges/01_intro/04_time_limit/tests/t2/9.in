925031077764215520107
