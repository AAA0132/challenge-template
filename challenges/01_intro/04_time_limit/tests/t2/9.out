427841247414813070417168561364088368405778
